### PR TITLE
Update SAR manual with specific steps to follow

### PR DESCRIPTION
**Pull request by benthorner**
https://trello.com/c/GZqSmo0H/1030-update-sar-requests-manual

Previously this was quite open-ended.

### CODE REVIEW

**Comment by benthorner** [comment]
Note: we're still trying to work out if there are more specifics steps to follow with regard to licensing.

File: source/manual/subject-access-requests.html.md [hunk 1]
```diff
@@ -13,24 +13,26 @@ Subject Access Requests (often referred to as a 'SAR') allow members of the
...
-### Others
+### Signon
 
-If any other information is requested, we should update this documentation so
-they become easier to deal with in the future.
+Check if the user [has an account](https://signon.publishing.service.gov.uk/users).
+
+### Licensing
+
+Ask in #govuk-licensing.
```
Comment (barrucadu) [inline source/manual/subject-access-requests.html.md:38]: When we hear back about the query it would be good to put it here

**Review by barrucadu — APPROVED** [review]
There will be some accounts stuff to add to this page too, but it's on our list

File: source/manual/subject-access-requests.html.md (hunk 1 above)
Comment (benthorner) [reply source/manual/subject-access-requests.html.md:38]: Indeed. I'm going to merge this is now, as plan to raise a PR when we know more about licencing.

**Pull request merged**